prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -109,63 +117,86 @@ func handlePullRequest(pc plugins.Agent, pre github.PullRequestEvent) error {
 	return handle(
 		pc.GitHubClient,
 		oc, pc.Logger,
-		pc.PluginConfig.Blunderbuss.ReviewerCount,
```

Comment:
I don't see the utility of changing this to pass the full config instead of the fields. A more minimal diff would keep it untouched

Reply:
I just noticed that we pass all the fields of the config struct and the number of arguments to `handle` is increasing. :grin:

I'll revert this back since it doesn't have a real impact.